src/gwt/tools/compile-themes.R

```diff
@@ -189,6 +211,7 @@ themeFiles <- list.files(
 ## Process the theme files -- we strip out the name preceeding the theme,
 ## and then add some custom rules.
 for (themeFile in themeFiles) {
+  str(themeFile)
```

Comment:
This is just here for debugging, I will remove it once https://github.com/rstudio/rstudio/issues/3907 is sorted.